backend/models/helper/model.js

```diff
@@ -503,7 +503,7 @@ function searchTree(account, model, branch, rev, searchString, username) {
 
 			});
 
-			return Ref.find({account, model}, {
+			return findRef(account, model, {
```

Comment:
This is very high coupling. This should be calling getRefNodes instead. ids to pass in is `history.current`. From what I can tell it's doing the same thing